packages/sdk/src/plt/v1/Token.ts

```diff
@@ -0,0 +1,254 @@
+import { ConcordiumGRPCClient } from '../../grpc/GRPCClient.js';
+import { AccountAddress, AccountInfo, TransactionHash } from '../../pub/types.js';
+import { AccountSigner } from '../../signHelpers.js';
+import { TransactionExpiry } from '../../types/index.js';
+import { bail } from '../../util.js';
+import { Token as GenericToken, holderTransaction, validateAmount, verify } from '../Token.js';
+import { TokenAmount, TokenId, TokenInfo } from '../index.js';
+import {
+    TOKEN_MODULE_REF,
+    TokenHolderOperation,
+    TokenOperationType,
+    TokenTransfer,
+    createTokenHolderPayload,
+} from './types.js';
+
+/**
+ * Enum representing the types of errors that can occur when interacting with PLT instances through the client.
+ */
+export enum TokenErrorCode {
+    /**
+     * Error representing an attempt transfer funds to an account which is either not on the token allow list, or is on
+     * the token deny list
+     */
+    NOT_ALLOWED = 'NOT_ALLOWED',
+    /** Error representing an attempt to transfer tokens from an account that does not have enough tokens to cover the
+     * amount */
+    INSUFFICIENT_FUNDS = 'INSUFFICIENT_FUNDS',
+}
```

Comment:
https://github.com/Concordium/concordium-grpc-api/pull/88/files#diff-a7337993ee30524fd2e3ee681d92ca3a483bb59389f312a741822917033175fdL864

Do we need the error case: 
- `non_existent_token_id`
- `invalid_account_reference`  when transferring to an account address that has not been created on chain.

Reply:
`Token.fromId` gets the token info from chain and will fail if it does not exist. Account checking is up to the consumer I'd say. We don't do this for any transactions.